webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -198,6 +204,7 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      showDeleteButton: !!this.contribution,
```

Comment:
The delete button should not be visible if there is no image. This happens if you edit a post which has no image:

<img width="1876" alt="Bildschirmfoto 2020-01-08 um 08 01 17" src="https://user-images.githubusercontent.com/25344101/71980448-53883880-3220-11ea-9c87-c414d53b1a8d.png">

And the delete button should not be visible if cropping is active:

<img width="1876" alt="Bildschirmfoto 2020-01-08 um 08 01 59" src="https://user-images.githubusercontent.com/25344101/71980595-b5e13900-3220-11ea-8179-2f1841acec9d.png">

And the delete button position should be the same as in the cropping, kind of.